### PR TITLE
Yandex/ssh communicator in export

### PR DESCRIPTION
**Pull request by Direnol**
This PR require #10317

### CODE REVIEW

**Comment by Direnol** [comment]
@GennadySpb

File: post-processor/yandex-export/cloud-init-script.go [hunk 1]
```diff
@@ -8,94 +8,51 @@ GetMetadata() {
...
+        sleep 5
+    done
+    echo "[${1}] not found"
+    return 1
 }
 
-INSTANCE_ID=$(GetInstanceId)
 PATHS=$(GetMetadata paths)
 S3_ENDPOINT="https://storage.yandexcloud.net"
+export AWS_SHARED_CREDENTIALS_FILE="/tmp/aws-credentials"
```
Comment (GennadySpb) [inline post-processor/yandex-export/cloud-init-script.go:32]: It's preferred to use tmpfs to store creds.

File: post-processor/yandex-export/step-create-s3-keys.go [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+	}
+	return multistep.ActionContinue
+}
+
+func (s *StepCreateS3Keys) Cleanup(state multistep.StateBag) {
+	driver := state.Get("driver").(yandex.Driver)
+	ui := state.Get("ui").(packer.Ui)
+
+	if val, ok := state.GetOk("s3_secret"); ok {
+		ui.Say("S3 secrets has been founded")
```
Comment (GennadySpb) [inline post-processor/yandex-export/step-create-s3-keys.go:54]:
```suggestion
		ui.Say("S3 secrets have been found")
```

**Comment by GennadySpb** [comment]
LGTM!

**Pull request merged**